### PR TITLE
Css docs

### PR DESCRIPTION
**Pull request by tewarimohit**

### CODE REVIEW

File: css_docs/drawer.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    font-weight: 800;
+}
+.subtext{
+    font-size: small;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    color: gray;
+}
+.drawer-elements{
+    position: relative;
```
Comment (tanishk15sharma) [inline css_docs/drawer.css:41]: use prettier

**Review by tanishk15sharma — APPROVED** [review]
Suggestion : do give a live link of your component or a screenshot or a video to let us  know what is happing in UI . 
use prettier for better code visibility

File: css_docs/drawer.css (hunk 1 above)
Comment (tewarimohit) [reply css_docs/drawer.css:41]: Sure Thanks..

**Comment by tewarimohit** [comment]
https://user-images.githubusercontent.com/89827782/152680797-bba3523d-04c4-482c-9a42-432220837883.mp4

**Pull request merged**